docs/user-guide.md

```diff
@@ -59,7 +59,7 @@ includes references that can be null. Code that deals with those references must
 be able to deal with the null case.
 
 ```java
-  public static void print(@Nullable String x) {
+  void print(@Nullable String x) {
```

Comment:
I'm wondering whether we should remove some of the boilerplate that isn't strictly needed.
Some methods are later called, so e.g. they would need to remain static. But maybe this could be simplified.

Reply:
OK, we can certainly remove many `public` and/or `static` modifiers.